### PR TITLE
Update Kubernetes deployments docs

### PR DESCRIPTION
**Pull request by happyRip**
#### Summary

References #1307 #1335

#### Screenshots

The Things Stack Upgrading

#### Changes

- Add `Upgrades` section to AKS docs.
- Update `Database Migrations` section of Generic k8s docs.

#### Notes for Reviewers

nil

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links. Test with `HUGO_PARAMS_SEARCH_ENABLED=true` if style changes will affect the search bar.
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

File: doc/content/the-things-stack/host/kubernetes/azure/upgrading/_index.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+weight: 4
+---
+
+This page describes the steps for upgrading {{% tts %}} on Azure Kubernetes Service.
+
+<!--more-->
+
+## Database Migration
+
+The Things Stack database entries are managed between minor versions using database migrations.
```
Comment (KrishnaIyer) [inline doc/content/the-things-stack/host/kubernetes/azure/upgrading/_index.md:13]: It's not the entities that are modified but the schema.

File: doc/content/the-things-stack/host/kubernetes/azure/upgrading/_index.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+{{< note >}}
+User are informed of required migrations for each version via the [Release Notes](https://www.thethingsindustries.com/docs/whats-new/). It’s mandatory to run required database migrations for The Things Stack to function properly.
+{{</ note >}}
+
+To migrate the database during an upgrade set the respectable Terraform variables to `true`.
```
Comment (KrishnaIyer) [inline doc/content/the-things-stack/host/kubernetes/azure/upgrading/_index.md:19]: `respective`.

File: doc/content/the-things-stack/host/kubernetes/generic/database-migrations/_index.md [hunk 3]
```diff
@@ -14,6 +14,28 @@ This page describes the steps for performing database operations on a {{% tts %}
 
 ## General Procedure
 
+Set the respectable Helm chart values to `true` to perform database migrations during Helm chart upgrade.
```
Comment (KrishnaIyer) [inline doc/content/the-things-stack/host/kubernetes/generic/database-migrations/_index.md:17]: `respective`

**Review by KrishnaIyer — APPROVED** [review]
Great. Thanks

**Review by KrishnaIyer — APPROVED** [review]

**Review by nejraselimovic — APPROVED** [review]

**Pull request merged**